web/ui-go/generated/dev/enola/core/enola_core.pb.go

```diff
@@ -0,0 +1,502 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
As this is generated code, when switching to a Bazel-based `BUILD`, I suspect it's possible to avoid having this in git, based on #227 ?